src/app/services/user.service.ts

```diff
@@ -36,18 +37,26 @@ export class UserService {
   }
 
   getUserByUuid(userUuid: string): Observable<User> {
-    return this.http.get<User>(this.api + userUuid, { headers: this.headers });
+    return this.http.get<User>(`${this.api}/${userUuid}`, { headers: this.headers });
+  }
+
+  getAddressByUserUuid(userUuid: string): Observable<Address> {
+    return this.http.get<Address>(`${this.api}/${userUuid}/addresses`, { headers: this.headers });
```

Comment:
should be changed this from api side also
```suggestion
    return this.http.get<Address>(`${this.api}/${userUuid}/address`, { headers: this.headers });
```